spec/searchmetrics_client/response_spec.rb

```diff
@@ -12,8 +12,9 @@
     include_context 'with api credentials'
 
     describe '#value' do
-      it 'return array data' do
-        expect(response.value).to be_instance_of Array
+      it 'return hash data' do
+        expect(response.value).to be_instance_of Hash
+        expect(response.value).to eq(balance: 59_981)
```

Comment:
This data ( `balance: 59_981`) does not look like a valid response to me. It is indeed the value the API returns now after this PR, but a valid response for the method `ResearchKeywordsGetListKeywordinfo` if we do a request like 
````
response = SearchmetricsClient::Request.send_request_from_hash(
             endpoint: 'ResearchKeywordsGetListKeywordinfo',
             params: { countrycode: 'PL', keyword: 'ruby' })
response.value
```
should look like
```
[{:cpc=>"2.37", :competition=>"3", :search_volume=>"12366", :trend=>"81:100:100:81:81:81:81:66:81:81:81:81", :ad_budget=>"29307.419999999998", :integration=>"images", :advertiser_count=>"0", :keyword=>"ruby", :year_month=>"201601"}]
```
This is what the old version in the master branch returns and what the API documentation says
http://api.searchmetrics.com/v3/documentation/api-calls/service/ResearchKeywordsGetListKeywordinfo
 
But now (if we use this branch) for API V3 it returns only a hash with a single value for `balance` like
```
{:balance=>52489}
```
This looks more like a response that our account has exceeded some kind of limit or "balance".